app/assets/stylesheets/friend_list.scss

```diff
@@ -0,0 +1,199 @@
+/**
+ * GMAIL Like friend css
+ * CREDITS: http://css-tricks.com/replicating-google-hangouts-friend/
+ *
+ */
+
+
+@media #{$medium-up} {
+    .friendbox {
+        position: fixed;
+        position: expression("absolute");
+        width: 280px;
+        right: 20px;
+        top: 0px;
+    }
+}
+
+
+@media #{$small-only} {
+    .friendbox {
+        position: fixed;
+        position: expression("absolute");
+        width: 280px;
+        display: none;
+        padding-bottom: 100px;
+        .fa {
+            font-size: 20px;
+        }
+    }
+}
+
+.friendboxhead {
+    background: #666;
+    color: white;
+    padding: 0.5rem;
+    overflow: hidden;
+
+    border-right: 1px solid rgba(85, 85, 85, 0.87);
+    border-left: 1px solid rgba(85, 85, 85, 0.87);
+}
+
+.friendboxhead h1 {
+    display: inline;
+    font-size: 17px;
+    font-weight: 700;
+    color: #fff;
+}
+
+.friendboxblink {
+    background-color: #176689;
+    border-right: 1px solid #176689;
+    border-left: 1px solid #176689;
+}
+
+.friendboxcontent {
+    font-family: arial, sans-serif;
+    height: 150px;
+    width: 280px;
+    overflow-y: auto;
+    padding: 7px;
+    border-left: 1px solid #cccccc;
+    border-right: 1px solid #cccccc;
+    border-bottom: 1px solid #eeeeee;
+    background: #e5e5e5;
+    line-height: 1.3em;
+    list-style: none;
+    display: none;
+    a {
+        margin-left: 30px;
+    }
+}
+
+.friendboxcontent li {
+    padding: 0.5rem;
+    overflow: hidden;
+    display: flex;
+}
+
+.friendboxcontent .avatar {
+    width: 40px;
+    height: 40px;
+    position: relative;
+}
+
+.friendboxcontent .avatar img {
```

Comment:
Merge rule `.friendboxcontent .avatar img` with rule on line 55<br>Selector should have depth of applicability no greater than 2, but was 3